pages/events/index.vue

```diff
@@ -31,19 +31,32 @@ import QiskitPage from '~/components/qiskit/QiskitPage.vue'
 
 export default class extends QiskitPage {
   routeName: string = 'events'
+  windowWidth: Number = 0
 
   autoplayVideo () {
+    if (!this.$refs.video) {
+      return
+    }
+
+    const video = this.$refs.video as any
+
     setTimeout(() => {
-      if (this.$refs.video) {
-        this.$refs.video.load()
-        this.$refs.video.muted = true
-        this.$refs.video.play()
-      }
+      video.load()
+      video.muted = true
+      video.play()
     }, 1)
   }
 
-  created () {
-    this.autoplayVideo()
+  get isDesktop () {
+    return this.windowWidth > 600
+  }
+
+  mounted () {
+    this.windowWidth = window.innerWidth
+
+    this.$nextTick(() => {
```

Comment:
You can replace this code with:
```js
await this.$nextTick()
this.autoPlay()
```
You will need to prefix `async` to the function name.

Reply:
Done 👍